src/scripts/website.sh

```diff
@@ -11,6 +11,13 @@ WEBSITE_SRC_DIR=./www-src
 rm -rf $WEBSITE_SRC_DIR $WEBSITE_DIR
 mkdir -p $WEBSITE_SRC_DIR
 
+# build manual as pdf for download
+sphinx-build -t website -c "$SPHINX_CONFIG" -b "latex" doc/manual latex
+cd latex
+pdflatex botan.tex
```

Comment:
I don't think running `pdflatex` once is sufficient, as it won't account for index and TOC.